src/com/projectkorra/projectkorra/waterbending/util/WaterReturn.java

```diff
@@ -148,42 +148,38 @@ public static boolean hasWaterBottle(Player player) {
 		if (inventory.contains(Material.POTION)) {
 			ItemStack item = inventory.getItem(inventory.first(Material.POTION));
 			PotionMeta meta = (PotionMeta) item.getItemMeta();
-			return meta.getBasePotionData().getType().equals(PotionType.WATER);
+			return meta.getBasePotionData().getType() == PotionType.WATER);

```

Comment:
It appears as though you have an extra/missing parenthesis here.

Reply:
Oh, thanks!